examples/client/client.c

```diff
@@ -182,6 +188,17 @@ static int ClientBenchmarkConnections(WOLFSSL_CTX* ctx, char* host, word16 port,
             if (ssl == NULL)
                 err_sys("unable to get SSL object");
 
+    #ifdef WOLFSSL_TLS13
+            if (!benchResume || !noPskDheKe) {
+                if (wolfSSL_UseKeyShare(ssl, WOLFSSL_ECC_SECP256R1)
```

Comment:
Changed to use a default unless user calls wolfSSL_NoKeyShare() in which case no KeyShare data is sent and the HelloRetryRequest message will be used to negotiate group.